### PR TITLE
Add winebottler.rb

### PR DESCRIPTION
**Pull request by cheyao**
- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.
- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

Everything works fine, tested by me.

### CODE REVIEW

**Comment by cheyao** [comment]
Burh I am on macos and the install works for me

File: Casks/winebottler.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask "winebottler" do
+  version "1.8.6"
+  sha256 :no_check
+
+  url "https://winebottler.kronenberg.org/combo/builds/WineBottlerCombo_1.8.6.dmg"
```
Comment (SMillerDev) [inline Casks/winebottler.rb:5]:
```suggestion
  url "https://winebottler.kronenberg.org/combo/builds/WineBottlerCombo_#{version}.dmg"
```

File: Casks/winebottler.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+cask "winebottler" do
+  version "1.8.6"
+  sha256 :no_check
```
Comment (SMillerDev) [inline Casks/winebottler.rb:3]: Why not?

File: Casks/winebottler.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  url "https://winebottler.kronenberg.org/combo/builds/WineBottlerCombo_1.8.6.dmg"
+  name "WineBottler"
+  desc "Run Windows-based Programs"
+  homepage "https://winebottler.kronenberg.org/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
+  end
```
Comment (SMillerDev) [inline Casks/winebottler.rb:13]: Can't it just check https://winebottler.kronenberg.org/combo/builds ?

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Comment by aaronliu0130** [comment]
Could we depend on wine instead of using the combo installer in the spirit of deduplication?

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @cheyao.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

**Comment by p-linnane** [comment]
The latest build is from December 2019...doesn't this make it an "unmaintained app" per [Rejected Casks](https://docs.brew.sh/Acceptable-Casks#rejected-casks)?

**Pull request closed without merging**